CMakeLists.txt

```diff
@@ -62,17 +57,17 @@ option(DEBUG_MMS_CLIENT "Enable MMS CLIENT printf debugging" OFF)
 #				 DEBUG_IED_CLIENT DEBUG_MMS_SERVER DEBUG_MMS_CLIENT)
 
 include_directories(
-    ${CMAKE_CURRENT_BINARY_DIR}/config
```

Comment:
I think this one should be kept. Otherwise the wrong config file will be used. CMake copies the generated stack_config.h into the build directory

Reply:
Oh yes. You are right. I will update the PR